app.js

```diff
@@ -26,7 +26,7 @@ app.patch("/api/articles/:article_id", patchArticleById);
 
 app.get("/api/articles", getAllArticles);
 
-app.get("/api/articles?topic", getAllArticles);
+app.get("/api/articles?topic=", getAllArticles);
```

Comment:
You don't need this here, you use the /api/articles endpoint so you would never get to this